### ISSUE TITLE
Disconnection when high latency between both peer 

### ISSUE THREAD

**Issue by dduquesne** [Haivision/srt#525]
Hello,

We are experimenting an issue.
We are working in a live usecase.
We have some SRT TX going bad.
The common point is the latency.
It seems to happen when both ends are geographically distant.
So the latency is higher than other well working SRT TX.
We feel that this kind of issue happens when latency goes above 80 ms.
It look likes the control layer decide to shutdown the CNX.
SRT_SOCKSTATUS is SRT_CLOSED just before termination.
It happens randomly from a time perspective (No specific frequency).
Is there any timeout somewhere at the control layer ?
I have been looking at the core source code but i have not found anything yet.

Many thanks for any feedback.

**Comment by ethouris** [comment]
Ok, sorry for the long disruption.

I'm not sure if I understand this "above 80ms". Do you mean, a "less" latency than that? We use default 120ms of latency and how it should be best shaped, it depends on the RTT in the network (4*RTT value is recommended, so this is for RTT=30ms).

The socket status might be not exactly reliable. Better would be if we can get at least the pcap, at best debug logs. What I may speculate as the reason is that the receiver does not purge the receiver buffer on time, we've seen cases like that, but to be sure as to what happened, I need some live data.

The only case when you might have any timeout is in the application. The stransmit application has something like 30s timeout for any transfer going through the link, if there are no data sent during that period, it exits. But there's nothing like that in the SRT library.

**Comment by dduquesne** [comment]
Hello ethouris,

Many thank for your reply. Sorry for the delay.
Indeed, i have finaly used the beta Wireshark version having the SRT dissector.
I made a capture simultaneously on both side and have waited an issue occurance.
It was realy helpful.
We were able to demonstrate that something wrong was happening on the network. The application layer (SRT) was not in cause.
It was a weird hardware bug on a FW acceleration card that was triggering new drop rules randomly on SRT ports. 
Realy weird and dificullt to find.
We are OK now.
